Source/gs/GSH_OpenGL/GSH_OpenGL.cpp

```diff
@@ -801,6 +801,13 @@ void CGSH_OpenGL::SetupBlendingFunction(uint64 alphaReg)
 		glBlendColor(0, 0, 0, static_cast<float>(alpha.nFix) / 128.0f);
 		glBlendFuncSeparate(GL_ONE, GL_CONSTANT_ALPHA, GL_ONE, GL_ZERO);
 	}
+	else if((alpha.nA == ALPHABLEND_ABD_CD) && (alpha.nB == ALPHABLEND_ABD_ZERO) && (alpha.nC == ALPHABLEND_C_FIX) && (alpha.nD == ALPHABLEND_ABD_CD))
+	{
+		//1221 -> Cd * (1 + Fix)
+		float bf = static_cast<float>(alpha.nFix) / 128.0f;
+		glBlendColor(bf, bf, bf, bf);
+		glBlendFuncSeparate(GL_DST_COLOR, GL_CONSTANT_COLOR, GL_ONE, GL_ZERO);
```

Comment:
Is this correct? Wouldn't that give this blending formulae?
`FinalColor = SourceColor*DestColor + ConstColor*DestColor?`

I don't think this blending mode can be emulated using the fixed blending functions. I think we should do something similar to what's done for 1201 temporarily.

Reply:
Alright, I checked in the frame debugger and this (GL_DST_COLOR, GL_CONSTANT_COLOR) works in this particular case because all source pixels are 1 (white) when the cone is rendered. It effectively becomes `FinalColor = DestColor * 1 + ConstColor * DestColor` which is the same as `Cd * (1 + Fix)`.

But if the source color is anything other than 1, it's going to give a wrong result.

To make this work in the general use case, I think a good option would be to have the fragment shader output 1 in `gl_FragColor` when this blending mode is selected since we know the source color is not used in the final output.

Reply:
I saw you working on this on a separate branch. As I lack the time to implement this right now - should I remove the commit, and you fix the blend mode on your branch?

Reply:
Yes please! The other commits are good, so, I will be able to merge the PR.

Reply:
Thanks!

Reply:
Done - you are welcome. Rebased onto master for good measure.